src/content/blog/you-probably-don't-need-eslint-config-prettier-or-eslint-plugin-prettier/index.mdx

```diff
@@ -0,0 +1,216 @@
+We even explicitly recommend against `eslint-plugin-prettier` in the [typescript-eslint formatting FAQs](https://typescript-eslint.io/linting/troubleshooting/formatting#eslint-plugin-prettier) and [typescript-eslint performance troubleshooting docs](https://typescript-eslint.io/linting/troubleshooting/performance-troubleshooting#eslint-plugin-prettier).
+
+If `prettier/prettier` is enabled in your ESLint configuration, the best step you can take is to remove it and uninstall the `eslint-plugin-prettier` package altogether.
+Failing that, you can disable the rule manually (`"prettier/prettier": "off"` under `"rules"`).
+
+## In Conclusion
+
+_Formatting_ and _linting_ are two separate concerns.
+Mixing the two can have negative impacts on the performance and understandability of your developer tooling.
+My standard repository template, [`create-typescript-app`](https://github.com/JoshuaKGoldberg/create-typescript-app), keeps the two explicitly separate.
+
+If your ESLint configuration references [`eslint-config-prettier`](https://github.com/prettier/eslint-config-prettier), I suggest you try to remove it from your config.
+You probably don't need it anymore.
+
+If your ESLint configuration references [`eslint-plugin-prettier`](https://github.com/prettier/eslint-plugin-prettier), I strongly recommend you instead enable Prettier separately from ESLint.
+Running the `prettier/prettier` rule can introduce significant performance pain to your project.
+
+No matter what tooling your ESLint configuration enables, if you haven't overhauled it in a few years, I strongly recommend:
+
+1. Making sure `"eslint:recommended"` is in your rule extensions
+2. If you're using TypeScript:
+   1. Making sure at least [`plugin:@typescript-eslint/recommended`](https://typescript-eslint.io/linting/configs#recommended) is enabled - or even better, [`plugin:@typescript-eslint/recommended-type-checked`](https://typescript-eslint.io/linting/configs#recommended-type-checked)
+   2. Checking the [`.eslintrc.cjs` from `create-typescript-app`](https://github.com/JoshuaKGoldberg/create-typescript-app/blob/main/.eslintrc.cjs), for any rules or plugins generally applicable to your project
+3. Checking [github.com/dustinspecker/awesome-eslint](https://github.com/dustinspecker/awesome-eslint) for any additional plugins relevant to your project
+
+Happy linting, everyone! 🎉
+
+### Acknowledgements
+
+Much appreciation to [Anisha Malde](https://www.anisha.dev) for suggesting I write an explainer in this area and helping ideate contents.
+Thanks!
+🙌
+
+Thanks also to [Simon Lydell](https://github.com/lydell), the original creator and a long-time maintainer of `eslint-config-prettier`, for reviewing the post and suggesting the `npx eslint-config-prettier` approach.
+
+We should also note that although I personally disagree with `eslint-config-prettier` and `eslint-plugin-prettier`, they are legitimately useful tools for many.
```

Comment:
A couple clarifications:

* I disagree with _using_ them _today_
* I'd still rather work in a project that uses them than a project that doesn't format at all